<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCCA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../IMAGENS/img.qrc">
    <normaloff>:/Tela principal/INSS.png</normaloff>:/Tela principal/INSS.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>261</width>
        <height>130</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>130</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="noFrame">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/imagens/usuario.qrc.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 3px solid rgb(86, 86 , 86);
	border-radius: 10px;
	background-color:  rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
QLineEdit:hover {
	border: 3px solid rgb(0, 0, 0);
}
QLineEdit:focus { 
	border: 3px solid rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Matr√≠cula</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
image: url(:/imagens/Senha.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 3px solid rgb(86, 86 , 86);
	border-radius: 10px;
	background-color:  rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
QLineEdit:hover {
	border: 3px solid rgb(0, 0, 0);
}
QLineEdit:focus { 
	border: 3px solid rgb(255, 255, 0);
}</string>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/imagens/Senha.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>35</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 3px solid rgb(86, 86 , 86);
	border-radius: 10px;
	background-color:  rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
QLineEdit:hover {
	border: 3px solid rgb(0, 0, 0);
}
QLineEdit:focus { 
	border: 3px solid rgb(255, 255, 0);
}</string>
         </property>
         <property name="placeholderText">
          <string>Confirme a Senha</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>221</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="noFrame_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(75, 75, 75);
	border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(75, 75, 75);
	border: 2px solid rgb(75, 75, 75);
}
QPushButton:pressed {
	background-color:rgb(255, 234, 0);
	border: 2px solid rgb(255, 240, 69);
	color: rgb(0,0,0);
}

</string>
         </property>
         <property name="text">
          <string>Cadastrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(75, 75, 75);
	border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(75, 75, 75);
	border: 2px solid rgb(75, 75, 75);
}
QPushButton:pressed {
	background-color:rgb(255, 234, 0);
	border: 2px solid rgb(255, 240, 69);
	color: rgb(0,0,0);
}

</string>
         </property>
         <property name="text">
          <string>Voltar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../IMAGENS/imagens.qrc"/>
  <include location="../IMAGENS/img.qrc"/>
 </resources>
 <connections/>
</ui>
